diff -urN graveman-0.3.11.orig/configure.in graveman-0.3.11/configure.in
--- graveman-0.3.11.orig/configure.in	2005-05-05 08:48:28.000000000 +0200
+++ graveman-0.3.11/configure.in	2005-05-05 08:48:56.000000000 +0200
@@ -241,7 +241,7 @@
 fi
 
 dnl nls support
-GRAVEMAN_LANG="cs de es fr it lt nl no pl pt_BR ru sv"
+GRAVEMAN_LANG="cs de es fr it lt nb nl pl pt_BR ru sv"
 AC_ARG_ENABLE(nls, , [
 if test "A$enableval" = "Ayes"; then
  	ALL_LINGUAS="$GRAVEMAN_LANG"
diff -urN graveman-0.3.11.orig/desktop/graveman.desktop.in graveman-0.3.11/desktop/graveman.desktop.in
--- graveman-0.3.11.orig/desktop/graveman.desktop.in	2005-05-05 08:48:28.000000000 +0200
+++ graveman-0.3.11/desktop/graveman.desktop.in	2005-05-05 08:50:45.000000000 +0200
@@ -12,7 +12,7 @@
 Name[pl]=Graveman
 Name[it]=Graveman
 Name[sv]=Graveman
-Name[no]=Graveman
+Name[nb]=Graveman
 GenericName=CD Creator
 GenericName[cs]=CD/DVD vypalovací program
 GenericName[de]=CD ersteller
@@ -25,7 +25,7 @@
 GenericName[pl]=Program do nagrywania płyty CD/DVD
 GenericName[it]=Creatore di CD
 GenericName[sv]=CD Skapare
-GenericName[no]=CD Brenner
+GenericName[nb]=CD Brenner
 Comment=Create and copy data and audio CD
 Comment[cs]=Vytváří a kopíruje datová a hudební CD/DVD
 Comment[de]=erstellt und kopiert Daten und Audio-CDs
@@ -38,7 +38,7 @@
 Comment[pl]=Pozwala tworzyć i kopiować płyty CD/DVD z danymi i muzyką
 Comment[it]=Crea e copia CD dati, audio e DVD
 Comment[sv]=Skapa och kopiera data och ljud CD
-Comment[no]=Lage og kopiere data og lyd CD/DVD
+Comment[nb]=Lage og kopiere data og lyd CD/DVD
 Exec=graveman
 Terminal=false
 Type=Application
