diff -aurN graveman-0.3.12-4.orig/Makefile.am graveman-0.3.12-4/Makefile.am
--- graveman-0.3.12-4.orig/Makefile.am	2005-05-28 02:34:01.000000000 +0200
+++ graveman-0.3.12-4/Makefile.am	2005-10-05 22:37:24.000000000 +0200
@@ -4,25 +4,4 @@
 
 EXTRA_DIST = autogen.sh
 
-## install-data-local:
-
-
-##dist-hook:
-##	if test -d pixmaps; then \
-##	  for pixmap in pixmaps/*; do \
-##	    if test -f $$pixmap; then \
-###	      cp -p $$pixmap $(distdir)/; \
-##	    fi \
-##	  done \
-##	fi
-##	if test -d glade; then \
-##	  for glade in glade/*; do \
-##	    if test -f $$glade; then \
-##	      cp -p $$glade $(distdir)/; \
-##	    fi \
-##	  done \
-##	fi
-
-
-
 ACLOCAL_AMFLAGS = -I m4
